<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTrimmer</class>
 <widget class="QMainWindow" name="ImageTrimmer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonPrevFrame">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNextFrame">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxFrame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_frame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelImage1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame1">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrame1">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveLine">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWriteLineFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadLineFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonPrevFrame</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>prevFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNextFrame</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>nextFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1057</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUpdate</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>updateImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFrame</sender>
   <signal>editingFinished()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>updateImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1014</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveLine</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>saveLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>1073</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonWriteLineFile</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>saveLineMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>999</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>967</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoadLineFile</sender>
   <signal>clicked()</signal>
   <receiver>ImageTrimmer</receiver>
   <slot>loadLineMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1034</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>1004</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>inputFileSelecter()</slot>
  <slot>nextFrame()</slot>
  <slot>prevFrame()</slot>
  <slot>updateImages()</slot>
  <slot>imageSelecter()</slot>
  <slot>saveLine()</slot>
  <slot>saveLineMatrix()</slot>
  <slot>loadLineMatrix()</slot>
 </slots>
</ui>
